files/chrono/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "chrono",
+  "repo": "wanasit/chrono",
+  "files": {
+    "include": ["chrono.js", "chrono.min.js", "chrono.map.json"]
```

Comment:
Well..not really. I just followed the example and it might be useful when debugging the min.js. 
If you suggest put only production needed files in JSDelivr, I will remove it.